source/configure/plugins-configuration-settings.rst

```diff
@@ -84,6 +84,25 @@ Automatic prepackaged plugins
 |                                                                                                                                                                                 | - Environment variable: ``MM_PLUGINSETTINGS_AUTOMATICPREPACKAGEDPLUGINS``        |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
 
+.. config:setting:: plugins-enableuploads
+  :displayname: Upload Plugin (Plugins - Management)
+  :systemconsole: Plugins > Plugin Management
+  :configjson: EnableUploads
+  :environment: N/A
+
+  - **true**:  Enables uploading of plugins from your local computer to your Mattermost server for all System Admins.
```

Comment:
this line isn't needed since it's included in the table

Reply:
line removed.